<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Linia" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>961</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="czysc">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>430</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;
</string>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border: 2px solid black;
color:black;</string>
    </property>
   </widget>
   <widget class="QSlider" name="Suwak2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>290</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border: 2px solid black;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="b2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b4">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>130</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>130</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b1">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;
</string>
    </property>
    <property name="text">
     <string>Pixel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cofnij">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>430</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>RYSOWANIE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>250</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>DO ELIPSY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>OPCJE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b5">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>Bezier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b6">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>BSpline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b7">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>320</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>FloodFill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b8">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>320</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font-weight: bold;
border: 2px solid black;
color:black;</string>
    </property>
    <property name="text">
     <string>ScanLine</string>
    </property>
   </widget>
   <widget class="QSlider" name="RedSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="GreenSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="BlueSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>550</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HueSlider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SaturationSlider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ValueSlider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="RedSpin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="GreenSpin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="BlueSpin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HueSpin">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>460</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SaturationSpin">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>500</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ValueSpin">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>540</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
text-align: center;</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlueLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="HueLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>460</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="SaturationLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="ValueLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Linia</class>
   <extends>QWidget</extends>
   <header>linia.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
